LICENSES_bundled.txt

```diff
@@ -300,4 +300,9 @@ License 3-Clause BSD
 Name: getLebedevSphere
 Files: scipy/integrate/_lebedev.py
 License 2-Clause BSD
-  For details, see scipy/integrate/_lebedev.py
\ No newline at end of file
+  For details, see scipy/integrate/_lebedev.py
+
+Name: pyprima
```

Comment:
```suggestion
Name: prima
```

pyprima is merely a subfolder of PRIMA, I think we should use the proper name here?